mesonbuild/modules/python.py

```diff
@@ -122,13 +122,15 @@ def __init__(self, python: 'PythonExternalProgram', interpreter: 'Interpreter'):
         assert isinstance(prefix, str), 'for mypy'
         self.variables = info['variables']
         self.suffix = info['suffix']
+        self.limited_api_suffix = info['limited_api_suffix']
         self.paths = info['paths']
         self.pure = python.pure
         self.platlib_install_path = os.path.join(prefix, python.platlib)
         self.purelib_install_path = os.path.join(prefix, python.purelib)
         self.version = info['version']
         self.platform = info['platform']
         self.is_pypy = info['is_pypy']
+
```

Comment:
Nit pick, but I am not sure this line change added anything.

Reply:
Nitpicks are very welcome!